src/command/template/queries.rs

```diff
@@ -0,0 +1,216 @@
+#![allow(clippy::all, warnings)]
```

Comment:
this is very interesting! it doesn't quite match the setup we have in `rover-client` for making requests so we'll likely want the initial implementation to match. However if we wanted to we could decide we wanted to migrate the whole codebase to this style of checking in the generated code as it may make hunting down the automated types a bit easier? Not sure.

Reply:
I figure if we like the style then we should migrate piecemeal so it doesn't become a massive effort. I'm hoping we can mix and match styles, since I definitely prefer being able to see the generated code and getting completions.

There is also the option of generating in `build.rs` but not checking the actual code in... I think that JetBrains IDEs will still let you navigate to the definition and get completions and such for that sort of code. Not sure about other editors though.